File: Python/P2.py
def main():
    fibs = [1,1]
    while fibs[len(fibs)-1] < 4000000:
        l = len(fibs)
        fibs.append(fibs[l-1] + fibs[l-2])
    evenFibs = [n for n in fibs if n % 2 == 0]
    print total(evenFibs)

def total(l):
    t = 0
    for i in l:
        t += i
    return t

if __name__ == '__main__':
    main()

File: P2.java
package OneHundred;
import java.util.ArrayList;
/**
* Write a description of class EvenFib here.
* 
* @author (your name) 
* @version (a version number or a date)
*/
public class P2
{
    public static void main(String [] args){
        ArrayList fibs = new ArrayList();
        fibs.add(1);
        fibs.add(2);
        int val = 0;
        while((int)fibs.get(fibs.size() - 1) < 4000000){
            fibs.add((int)fibs.get(fibs.size() - 2) + (int)fibs.get(fibs.size() - 1));
        }
        for(int i = 0; i < fibs.size(); i++){
            if((int)fibs.get(i) % 2 == 0){
                val += (int)fibs.get(i);
            }
        }
        System.out.println(val);
    }
}
